src/trivia/util.h

```diff
@@ -570,6 +570,41 @@ is_exp_of_two(unsigned n)
 	return (n & (n - 1)) == 0;
 }
 
+static inline char **
+str_split(char *str, const char *delimiter, unsigned *size, bool copy)
```

Comment:
What's happening here? How it is different from basic strtok()? Anyway it lacks comments and tests.